src/shared/utils/assert-totals-are-reasonable.js

```diff
@@ -0,0 +1,12 @@
+const assert = require('assert')
+
+/**
+ * Sometimes folks don’t sum numbers properly, so give them 10% slack.
+ */
+const assertTotalsAreReasonable = ({ computed, scraped }) => {
```

Comment:
Nice, so this can be used to make sure the values match what we've added up.

Reply:
Yup. Was originally `===` matching but turns out people can't sum for crap 😂